project/src/common/const.ts

```diff
@@ -22,11 +29,26 @@ const Setting = {
   PlaceAmount: 132,
 } as const;
 
+enum CardType {
+  Favorites = 'favorites',
+  Cities = 'cities'
+}
+
+enum photoSize {
```

Comment:
```suggestion
enum PhotoSize {
```
Слишком абстрактно. Какой именно фото, нужно уточнить